Patches/Alpha Animals/ThingDefs_Races/AlphaAnimals_CE_Patch_Race_Demolisher.xml

```diff
@@ -70,7 +72,18 @@
 					</tools>
 				</value>
 			</li>
+
+			<!-- Make it spawn with ammo -->
 			
+			<li Class="PatchOperationAdd">
+				<xpath>/Defs/ThingDef[defName="AA_Demolisher"]/comps</xpath>
+				<value>
+					<li>
+						<compClass>CombatExtended.CompPawnGizmo</compClass>

```

Comment:
Honestly, we could probably just nuke the `AmmoUser` on it's just and save ourselves the trouble.